tools/js-sdk-release-tools/src/xlc/codeUpdate/updateUserAgent.ts

```diff
@@ -23,6 +24,13 @@ export function updateUserAgent(packageFolderPath: string, packageVersion: strin
             })
             break;
         case SDKType.ModularClient:
+            // Check if it's a model-only package for MLC
+            const isMlcModelOnly = await isModelOnly(packageFolderPath);
+            if (isMlcModelOnly) {
+                logger.info(`MLC package in ${packageFolderPath} is a model-only package, skipping user agent update`);
+                break;
```

Comment:
@MaryGao  @kazrael2119 wanna confirm that no client file is generated and no need to update user agent?

Reply:
yes, for model only case no client generated.